images/rook/Dockerfile

```diff
@@ -16,5 +16,16 @@ FROM BASEIMAGE
 
 ADD rook rookflex /usr/local/bin/
 
+# Setup user for build execution and application runtime
+ENV ROOK_ROOT=/
```

Comment:
@bassam I've used `/  here as `dataDirHostPath` is not known at this point. So I can't precisely allocate the right permissions to the appropriate directory.